CMakeLists.txt

```diff
@@ -112,8 +117,10 @@ if (INSTALL)
       set(LIBRARY_BASE ${INSTALL})
       # This is a terrible hack, but hey it works
       set(CMAKE_BUILD_RPATH "$ORIGIN:$ORIGIN/..")
-else()
+elseif(WRAP_PYTHON)
       set(MODULE_BASE ${PROJECT_BINARY_DIR}/neml)
+      set(LIBRARY_BASE ${PROJECT_BINARY_DIR}/neml)
```

Comment:
is `set(LIBRARY_BASE ${PROJECT_BINARY_DIR}/neml)` correct for `WRAP_PYTHON==True`

Reply:
Yep.